src/main/java/com/hillel/items_exchange/dto/UserDto.java

```diff
@@ -0,0 +1,40 @@
+package com.hillel.items_exchange.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.*;
+import java.time.LocalDate;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class UserDto {
+    private Long id;
+    @NotEmpty(message = "Username has to be not empty")
+    @Size(min = 3, max = 50, message = "Username has to be between 3 and 50 symbols")
+    private String username;
+    @NotEmpty(message = "Password must be not empty")
+    private String password;
+    @Email(message = "E-Mail has to be valid")
```

Comment:
Please add here:
@NotEmpty(message = "Email field is required")
@Email(message = "Please enter valid email address (Ex: username@example.com)")
@Size(max = 129, message = "Email length should be less than 129 characters. Please enter valid email address")